pkg/config/con.go

```
@@ -0,0 +1,947 @@
+	path string,
+	v *viper.Viper,
+) (bool, string, error) {
+	reader, err := os.Open(path)
+	if err != nil {
+		return false, "", err
+	}
+
+	defer func(reader *os.File) {
+		err := reader.Close()
+		if err != nil {
+			u.LogWarning(atmosConfig, fmt.Sprintf("error closing file '"+path+"'. "+err.Error()))
```

golangci-lint (1.62.2)

printf: non-constant format string in call to fmt.Sprintf

(govet)